root/utility/invariant.js

```diff
@@ -22,3 +22,8 @@ export default function invariant(
     throw new Error(msg);
   }
 }
+
+export function expect<T>(value: ?T): T {
+  invariant(value != null, 'Expected a non-null value');
+  return value;
+}
```

Comment:
This small little utility allowed me to avoid a bunch of ugly `?.`, and nest some calls (avoiding temporary assignments/if statements) for `tree.find` above, which normally can return `null` if an item isn't found in the list, but here we know they exist -- just have to convince Flow.

Reply:
Maybe `nonnull` is a better name (but I'd like to avoid something overly long).